### ISSUE TITLE
`make configint` results in test failure

### ISSUE THREAD

**Issue by zaksoup** [cloudfoundry-incubator/bosh-google-cpi-release#182]
Hi Folks,

We're running integration tests locally and see the following failure: 
```
  executes the VM lifecycle with a region backend service [It]
  /Users/pivotal/workspace/bosh-google-cpi-release/src/bosh-google-cpi/integration/vm_test.go:733

  Expected
      <*dispatcher.ResponseError | 0xc4203ab170>: {
          Type: "Bosh::Clouds::VMCreationFailed",
          Message: "VM failed to create: Failed to add instance \"https://www.googleapis.com/compute/beta/projects/cf-bosh-cpi-dev/zones/us-central1-a/instances/vm-06e26e99-18d9-451a-5806-a946c586c111\" to Backend Service \"cfintegration\"'s instance group named \"cfintegration-ilb\": Failed to add Google Instance 'vm-06e26e99-18d9-451a-5806-a946c586c111' to Google Instance Group 'cfintegration-ilb': googleapi: Error 400: Resource 'projects/cf-bosh-cpi-dev/zones/us-central1-a/instances/vm-06e26e99-18d9-451a-5806-a946c586c111' is expected to be in the network 'projects/cf-bosh-cpi-dev/global/networks/cfintegration' but is in the network 'projects/cf-bosh-cpi-dev/global/networks/cfintegration-custom'., wrongNetwork",
          CanRetry: true,
      }
  to be nil

  /Users/pivotal/workspace/bosh-google-cpi-release/src/bosh-google-cpi/integration/assertions.go:27
```

We believe this is due to a slight difference between how `make configint` behaves and how the `setup-infrastructure.sh` concourse task behaves:

In `make configint` we create a cfintegration backend service: https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/blob/a3daf0afc8c78aea50eaf21019e99790f0012ff3/src/bosh-google-cpi/Makefile#L142-L146
But there is already a backend service with this name created earlier in the same make task: https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/blob/a3daf0afc8c78aea50eaf21019e99790f0012ff3/src/bosh-google-cpi/Makefile#L106-L110

In `setup-infrastructure.sh` we create these backend services with **different** names: https://github.com/cloudfoundry-incubator/bosh-google-cpi-release/blob/a3daf0afc8c78aea50eaf21019e99790f0012ff3/ci/tasks/setup-infrastructure.sh#L52-L64

We'd like to do a focused PR to fix this test specifically but we are additionally interested in submitting a PR that will transition the current `gcloud` based scripts to terraform. Do you have any objections to this?

Ultimately the change we'd like to make is to have locally run integration tests consume info about the infrastructure in a uniform way with the concourse-run integration tests. Terraform provides a convenient way of enabling this with "outputs" but I'd love to have a deeper discussion if you'd prefer not to switch.

best,
Zak & @cunnie

**Comment by johnsonj** [comment]
Thanks for digging into this @zaksoup @cunnie. Terraform would be ideal and I have no objection to it. 
 The integration scripts were built before we embraced it. That's the only reason I can see for it not being used.

Happy to review the focused PR in addition to the bigger followup.

**Comment by zaksoup** [comment]
@johnsonj Awesome thanks. We just pushed #184 and will followup with a reconfiguration that uses Terraform. We're still running the full integration suite locally to confirm our changes didn't break anything else and will update the PR if we find any problems. Thanks!

**Comment by zaksoup** [comment]
Closed with merge of #184
